ui/src/components/Query.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { InputBar } from 'components/library/InputBar'
+import { ButtonColors, ButtonSizes, InputTypes } from './library/Enums'
```

Comment:
please use fullpath or relative path, not both.


nit: I would prefered full path